content/en/docs/tutorials/stateless-application/guestbook-logs-metrics-with-elk.md

```diff
@@ -43,6 +43,12 @@ Additionally you need:
 
 <!-- lessoncontent -->
 
+## Logging
+Generally, application logs are sent to the STDOUT and STDERR streams  and are configured by a logging driver. In the case of Kubernetes, logs are assigned via the kubelet service that runs on the node. You can have a look at the logging architecture(/docs/concepts/cluster-administration/logging) for detailed information 
```

Comment:
Hi @Aut0R3V . Here is some feedback:
- Is the goal to update the tutorial by adding an overview of k8s logging and modify the example to use DaemonSets?
- Some style edits:
   - You could add a newline between a heading and the following text.
   - To link to another page, write a Markdown style link as:

```
To learn about Kubernetes basic logging, node-level logging agents, and cluster-level logging, see [logging architecture](/docs/concepts/cluster-administration/logging/).

A [DaemonSet](/docs/concepts/workloads/controllers/daemonset/) creates a ...
```